_posts/2023-05-31-strategy-is.md

```diff
@@ -0,0 +1,75 @@
+
+## Creating the Department of Decentralized Digital Culture (a.k.a, that art/policy thing we’re doing)
+
+Over the past year, Hypha has acquired worker-members with interest and skills in a cross-cutting area that touches on decentralized tools, art, culture, advocacy, and collaboration. Hypha occupies a unique position of delivering both the technical skills in a nascent space (i.e. Web3 or DWeb), and also having a small r research understanding of the work and how to make it accessible. In other words, we can think and do. This nebulous mix is still in the framing stages as a business offering per se, but it’s useful to highlight the motivations that are driving us to expand our work in this corner of the interwebs. 
+
+**Convivial tooling:** “A tool is convivial if people have access to the design and knowledge needed to create it; if it allows creative adaptation to one’s own circumstances; and if it’s appropriate in the specific local context.” Inspired by Ivan Illich, this concept from the awesome book Free, Fair and Alive: The Insurgent Power of the Commons can be distilled to a desire to empower the people we collaborate with in our ongoing work. Yes, it means working with free and open source tools, but it also means scaffolding the necessary skill to allow for effective use of said tools (as an aside, there is a lot Web3 community builders can learn from the field of community informatics, but that’s a post for another day. But this is also, in part, where we bring a policy lens to our work.)
+
+**Imaginative solutions:** For us, creativity is paramount. We’re artists and writers, and we don’t leave this behind when the work gets technical. We’re still figuring out what this means in practice and finding the balance between budgets and beauty, and how to make projects sing while working as a small team. This goes beyond making nice things; a thoughtful and empathetic process of design and discovery are central to how we work.
+
+**Deep knowledge:** the tech sector has a notoriously short-term memory. Everything is innovative, everything is cutting-edge if you can’t remember whose shoulders you’re standing on. We like to think that we bring a more nuanced and informed perspective to bear; collectively, we have decades of experience to draw on, helping us see where technology is going by understanding where we’ve been. 
+
+The ideal projects we’d tackle as part of the cheekily named D3C are with digital savvy, mid-to-large arts and culture organizations who are interested in pushing the boundaries of how they use and interact with technology; that’s a pretty wide-open field, but we’ll refine this as we go. Everything is iterative.
+
```

Comment:
```suggestion

---
```

Would a line break that separates the summary from the previous section be useful?